src/libraries/System.Threading.Tasks.Extensions/tests/ValueTaskTests.cs

```diff
@@ -1168,6 +1168,7 @@ public void Generic_ToString_Success()
 
         [Theory]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/50722", typeof(PlatformDetection), nameof(PlatformDetection.IsBuiltWithAggressiveTrimming), nameof(PlatformDetection.IsBrowser))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/50722", typeof(PlatformDetection), nameof(PlatformDetection.IsAndroidAOT))]
```

Comment:
Same, as for TagToTokenTests.cs, if the failure is the same and known for same underlying reason, can the issue be updated to include that it fails on Android AOT too?

Reply:
same, added a comment